apps/api/src/coscrad-cli/execute-command-stream.cli-command.ts

```diff
@@ -371,6 +371,16 @@ export class ExecuteCommandStreamCliCommand extends CliCommandRunner {
                 const newEntries = (
                     fsaToExecute.payload as ImportEntriesToVocabularyList
                 ).entries.map((entry) => {
+                    if (
+                        ![APPEND_THIS_ID, GENERATE_THIS_ID].some((prefix) =>
+                            entry.termId.includes(prefix)
+                        )
+                    ) {
+                        // TODO add a regression test
```

Comment:
Do this.

Reply:
I added a test for the case of using all real UUIDs. But this didn't repro the issue. The only thing I can think is that it had to do with the size of the given `entries` array in the real data. But I'll leave this be for now.